src/components/rating/Rating.tsx

```diff
@@ -52,15 +52,13 @@ const Rating = ({ isResponseGenerated, feedbackId }: RatingProps) => {
     if (feedback.rating === Ratings.NEGATIVE && !feedback.answerQuality) {
       return;
     }
-    const { status, error } = await SupaBaseDatabase.getInstance().addFeedback({
-      ...feedback,
-      feedbackId,
-    });
 
-    if (status >= 200 && status < 300 && !error) {
-      setIsFeedbackSubmitted(true);
-      console.log("Feedback sent successfully");
+    if (isSupabaseInitialized) {
+      await addFeedback(feedback, feedbackId);
```

Comment:
Since this function returns true or false, we should assign the return value a variable and check if the feedback was submitted. Only then should we `setFeedBackSubmitted` to true

Reply:
yeah, this is a hack to get the page to close.